debian/rules

```diff
@@ -17,27 +21,32 @@ override_dh_auto_configure:
 	                     CONFIG+=nodoc \
 	                     CONFIG+=enable-presage \
 	                     CONFIG+=enable-hunspell \
-	                     CONFIG+=enable-pinyin
+	                     CONFIG+=enable-pinyin \
+	                     $(CONFIGURE_OPTS)
 
 override_dh_auto_test:
-	python3 -m flake8 .
-ifeq ($(DEB_BUILD_GNU_TYPE),$(DEB_HOST_GNU_TYPE))
-	# Tests write $HOME/.presage/lm.db, make sure $HOME is
-	# writable.
-	rm -rf $(CURDIR)/debian/test-home
-	mkdir -p $(CURDIR)/debian/test-home
-	HOME=$(CURDIR)/debian/test-home xvfb-run -a make check
+ifeq (,$(findstring nocheck,$(DEB_BUILD_OPTIONS)))
+    python3 -m flake8 .
+    ifeq ($(DEB_BUILD_GNU_TYPE),$(DEB_HOST_GNU_TYPE))
+        # Tests write $HOME/.presage/lm.db, make sure $HOME is
+        # writable.
+        rm -rf $(CURDIR)/debian/test-home
+        mkdir -p $(CURDIR)/debian/test-home
+        HOME=$(CURDIR)/debian/test-home xvfb-run -a make check
+    endif
 endif
 
 override_dh_clean:
 	rm -rf $(CURDIR)/debian/test-home
 	dh_clean
 
 override_dh_install:
+ifeq (,$(findstring nocheck,$(DEB_BUILD_OPTIONS)))
```

Comment:
This is not 100% correct: the code here is installing files, not running tests. I think we should be running this in any case. If you do that change I suggest in `debian/control`, you'll see that this works fine.

Reply:
oops